src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Image
+
+A widely supported way of uploading images to an image CDN is to send them via an HTTP POST request to the image CDN’s API. However, manually uploading images is not always necessary in order to use an image CDN. Some image CDNs can be configured to automatically retrieve images from their existing locations when they are needed.
+
+This is often achieved by including the complete URL of the _existing image_ within the URL for the image generated by the image CDN. For example, you might see a URL that looks like this: `https://[my-site.example-cdn.com/https://flowers.com/daisy.jpg/quality=auto`. This URL would retrieve and optimize the image that exists at `https://flowers.com/daisy.jpg`.
+
+
+### Security key
+
+A security key prevents other people from creating new versions of your images. If this feature is enabled, each new version of an image requires a unique security key. If someone tries to change the parameters of the image URL but doesn’t provide a valid security key, they won’t be able to create a new version. Your image CDN will take care of the details of generating and tracking security keys for you.
+
+
+### Transformations
+
+Image CDNs offer tens, and in some cases hundreds, of different image transformations. These transformations are specified via the URL string and there are no restrictions on using multiple transformations at the same time. In the context of web performance, the most important image transformations are size, pixel density, format, and compression. These transformations are the reason why switching to an image CDN typically results in a significant reduction in image size.
+
+There tends to be an objectively “best” setting for performance transformations, so some image CDNs support an “auto” mode for these transformations. For example, instead of `format=webp` you would include `format=auto` in the image’s URL string. This setting would leave it up to the CDN to determine the best format to serve to a client. Signals that an image CDN can use to determine the best way to transform an image include:
+
+
+
+*   [Client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints) (e.g. Viewport-Width, DPR, and Width)
+*   [Save-Data](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Save-Data) header
+*   [user-agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) header
+*   [NetworkInformation API](https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API)
+
+For example, the image CDN might serve JPEG XR to an Edge browser, WebP to a Chrome browser, and JPEG to a very old browser. “Auto” settings are popular because they allow you to take advantage of image CDNs’ significant expertise in optimizing images without you needing to make code changes in order to adopt new technologies once they’re supported by the image CDN.
```

Comment:
```suggestion
For example, the image CDN might serve JPEG XR to an Edge browser, WebP to a Chrome browser, and JPEG to a very old browser. "Auto" settings are popular because they allow you to take advantage of image CDNs’ significant expertise in optimizing images without you needing to make code changes in order to adopt new technologies once they’re supported by the image CDN.
```